src/identity/actions/thunks/index.ts

```diff
@@ -81,12 +84,18 @@ export const getCurrentOrgDetailsThunk =
 
       dispatch(setCurrentOrgDetails(orgDetails))
       dispatch(setCurrentOrgDetailsStatus(RemoteDataState.Done))
+
+      return orgDetails
     } catch (err) {
       dispatch(setCurrentOrgDetailsStatus(RemoteDataState.Error))
 
+      const state = getState()
+      const identity = selectCurrentIdentity(state)
+
       reportErrorThroughHoneyBadger(err, {
         name: 'Failed to fetch /quartz/orgs/:orgId',
-        context: {state: getState()},
+        context: {identity},
       })
+      throw new Error(err)
     }
```

Comment:
Modifies this thunk to throw an error to the UI that can be caught to deliver a notification.

Reduced the amount of data we're sending on honeybadger errors because it's oversize and unnecessary. The only information we really need to debug an api failure here is the user's identity.